server.go

```diff
@@ -59,6 +65,12 @@ type Server struct {
 	// is added.
 	responses chan processedRequest
 
+	// Number of maximum retries if a requet faies to process before stop
```

Comment:
Spelling :)

Reply:
No need to fix if change to middleware, then I'll just delete this branch :)